src/Avalonia.Base/Platform/IRuntimePlatform.cs

```diff
@@ -13,15 +13,17 @@ public interface IRuntimePlatform
     public record struct RuntimePlatformInfo
     {
         public FormFactorType FormFactor => IsDesktop ? FormFactorType.Desktop :
-            IsMobile ? FormFactorType.Mobile : FormFactorType.Unknown;
+            IsMobile ? FormFactorType.Mobile : IsTv ? FormFactorType.Tv : FormFactorType.Unknown;
         public bool IsDesktop { get; set; }
         public bool IsMobile { get; set; }
+        public bool IsTv { get; set; }
     }
 
     public enum FormFactorType
     {
         Unknown,
         Desktop,
-        Mobile
+        Mobile,
+        Tv
```

Comment:
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

I'm thinking `TV` since it's two letter abbreviation. You can make an argument like `Pi` and `Id` though.

Reply:
That convention is a bit confusing. If I understand correctly, `IO` and `Id` are different because the former corresponds to two words, while the latter corresponds to a single word. So `Tv` would be the correct capitalization, I think.

I'm a bit biased because I would write `Io` anyway. I honestly don't understand the difference between abbreviations with two or more letters, as the readability is the same.